### PR TITLE
[Modular] More language choices for specific species

### PR DESCRIPTION
**Pull request by desertmote**
## About The Pull Request

Freeing up species restricted languages for species that could possibly speak them.

- Humans can learn moffic and draconic
- Synths of any kind can learn monkey, slime, nekomimetic, xenoknockoff, moffic, buzzwords and draconic.
- Slimes can learn monkey, nekomimetic, xenoknockoff, moffic, buzzwords and draconic.
- Moths can learn buzzwords.
- Anthro Insects can learn moffic and buzzwords.
- Anthro mammals can learn monkey, nekomimetic, xenoknockoff and moffic.
- Humanoids can learn nekomimetic, xenoknockoff, moffic and draconic.

## Why It's Good For The Game

My rationale for these species combined with these languages: If the species can wear the body-parts of another species, they can opt to learn the language. A humanoid who is practically a felinid should be able to speak or understand nekomimetic. A mammal who has moth wings should be able to speak or understand moffic, et cetera, et cetera.

It also makes the language points and linguist perk useful.

**Please leave recommendations for additions or changes in the comments,** this should be updated with a couple of commits, as I must have missed something.

## Changelog
:cl:
add: Humans can learn moffic and draconic
add: Synths of any kind can learn monkey, slime, nekomimetic, xenoknockoff, moffic, buzzwords and draconic.
add: Slimes can learn monkey, nekomimetic, xenoknockoff, moffic, buzzwords and draconic.
add: Moths can learn buzzwords.
add: Anthro Insects can learn moffic and buzzwords.
add: Anthro mammals can learn monkey, nekomimetic, xenoknockoff and moffic.
add: Humanoids can learn nekomimetic, xenoknockoff, moffic and draconic.
/:cl:

### CODE REVIEW

**Comment by ZephyrTFA** [comment]
https://github.com/Skyrat-SS13/Skyrat-tg/pull/7367

**Comment by ZephyrTFA** [comment]
Again. This is in the middle of a feature freeze.

**Comment by desertmote** [comment]
Is that supposed to stop me from creating pull requests? What’s wrong with waiting to merge later, this change is not just tentative but also requires discussion. It would be a waste of time to sit out the freeze before doing so.

**Comment by Guidesu** [comment]
> Again. This is in the middle of a feature freeze.

I don't know but gandalf merging Nanotrasen Fleet Expansion and several stuffs at feature freezes says otherwise but i dunno.

File: tgstation.dme [hunk 1]
```diff
@@ -3875,7 +3875,6 @@
 #include "modular_skyrat\master_files\code\modules\events\spider_infestation.dm"
 #include "modular_skyrat\master_files\code\modules\jobs\departments\departments.dm"
 #include "modular_skyrat\master_files\code\modules\jobs\job_types\cyborg.dm"
-#include "modular_skyrat\master_files\code\modules\language\language_holders.dm"
```
Comment (desertmote) [inline tgstation.dme:3878]: About this, we have a more populated file located here:
\modular_skyrat\modules\customization\modules\language\language_holder.dm

**Comment by SpaceVampire** [comment]
This really ruins the whole point of species languages if anyone can learn them. One of the core parts is being able to look at someone and figure out if they have a good chance of understanding you or not. Just having some moth wings doesn't make you a moth in regards to any of the other mechanical features, it is just a cosmetic choice.

**Comment by desertmote** [comment]
> Just having some moth wings doesn't make you a moth in regards to any of the other mechanical features, it is just a cosmetic choice.

It's not about the "cosmetic". It doesn't depend on what mutant bodypart they have, it's about if they used **two of their three** linguistic points to learn to speak moffic. If someone does, then surely they have proper IC reasoning for it.
This change doesn't take away anything, it allows very restricted access to a language that is suitable with your character.

Should humans really be allowed moth wings, but not be allowed to spend, again TWO of THREE linguistic points to speak moth, too?

**Comment by ZephyrTFA** [comment]
> > Just having some moth wings doesn't make you a moth in regards to any of the other mechanical features, it is just a cosmetic choice.
> 
> It's not about the "cosmetic". It doesn't depend on what mutant bodypart they have, it's about if they used **two of their three** linguistic points to learn to speak moffic. If someone does, then surely they have proper IC reasoning for it.
> This change doesn't take away anything, it allows very restricted access to a language that is suitable with your character.
> 
> Should humans really be allowed moth wings, but not be allowed to spend, again TWO of THREE linguistic points to speak moth, too?

It does not make sense for them to speak Moffic if they aren't a moth or some form of moth no.

**Comment by desertmote** [comment]
It's very anal that our character customizer allows for extreme hybrid characters, but prevent them from speaking or understanding language they take part in, even with the framework to allow proper restriction existing.

I personally play a humanoid who is 2/3 felinid, but I cannot let them understand or speak it. This is literally a customization failure and it should go through fixing.

**Comment by Snakebittenn** [comment]
skyrat isn't that kind of server.
what i mean by this is that our races are not explicitly, like. 'this race is this race and this race has this culture and this biology.'
our species are more like guidelines than hard biologies and we use them for all kinds of wacky, crazy shit. it's fully possible to play an anthromorph (with sprites) that's meant to be some kind of hybrid of a xenomorph, or a human that's like one of the failed ripley clones, or even (the most light example here) an IPC that was purpose-built for engaging with xenomorphs and hybrid xenomorph organisms.

or, for draconic, you could play a hybrid of a lizard and something else, or a slime that's meant to originally have been a lizard (and for all intents and purposes looks like a translucent lizard), or a human that's spent decades learning to speak their language (with assistance), whatever.

yet, currently, all of those things right now are limited to 'you have to mechanically be a xenohybrid/lizard to speak this' and there's no way to even pick a culture that can do that for you.(edited)
we have /way/ more customization in our various species than, say, tg. some people make those species into things that barely even resemble their original one.
i know for instance several chars that have plenty of buildup to justify knowing draconic, but making them have to ahelp every single round (or run down to lavaland to get the draconic necklace and run back to the station) is kind of silly. i would much rather trade draconic being a 'species specific language' away to further encourage character creativity.

i'll admit, i will eyeroll abut people speaking xenoknockoff or whatever as a normal green-haired janitor girl and their justification is 'don't think about it too hard' or whatever but it'll be more than worth it to enable people who /are/ creative and think deeply about these things to be able to further express who their character is.

**Comment by FlamingLily** [comment]
> Again. This is in the middle of a feature freeze.

Feature freezes don't exist and you know this

**Comment by Vizoplez** [comment]
I'm going to have to vote no.


As an ex-developer this is a poor idea for several reasons.

For one, while this may seem like an outwardly good idea, several languages involved cannot actually be spoken, realistically, by other species (Such as moffic being spoken by humans), due to its inherent usage of wings, clicks and other sounds usually not able to be produced by humans.

For two, many of the languages I believe are just absolute bloat, and some are just memes. The felinid language needs an outright replacement and not more races being able to use it (key. Several keywords that are extremely meme being in the language dictionary.

Anyway, this is a bad idea.

**Comment by Gandalf2k15** [comment]
After giving it careful consideration I don't think this is needed and actually takes away from the uniqueness of species. So we're gonna pass on this one.

**Pull request closed without merging**